tests/testthat/test-printing.R

```diff
@@ -545,35 +545,19 @@ test_that("row label indentation is kept even if there are newline characters",
 })
 
 test_that("Support for newline characters in all the parts", {
-    set.seed(1)
-    DM_trick <- DM %>% 
-        mutate(ARM2 = sample(c("TWO\nwords\n", "A wo\n\nrd"),
-                             replace = TRUE, nrow(DM))) # last \n is eaten up
-    levels(DM_trick$SEX)[3] <- "U\nN\nD\n"
-    tbl <- basic_table() %>% 
-        split_rows_by("SEX", split_label = "m\nannaggia\nsda\n",
-                      label_pos = "visible") %>% # last \n bug
-        split_cols_by("ARM2", split_label = "sda") %>% 
-        analyze("BMRKR1", na_str = "asd\nasd") %>%  # \n error
-        build_table(DM_trick)
-    
-    top_left(tbl) <- c("\na", "b\nd\n\n", "c\n\n") # last \n is eaten up, if in the middle error
-    main_title(tbl) <- "why not \nalso here\n"
-    out <- strsplit(toString(tbl, hsep = "-"), "\\n")[[1]]
+    out <- strsplit(toString(tt_for_nl, hsep = "-"), "\\n")[[1]]
     expected <- c(
-        "why not ",
+        "why not",
         "also here",
         "",
-        "",
         "---------------------------------",
         "                                 ",
-        "a                                ",
-        "b                                ",
-        "d                                ",
         "                                 ",
-        "                             A wo",
-        "c                     TWO        ",
-        "                     words    rd ",
+        "                                 ",
+        "a                                ",
+        "b                            A wo",
+        "d                     TWO        ",
+        "c                    words    rd ",
```

Comment:
topleft is at the moment squashed, it will need a bit of revision (if spaces are added beside "\n") it keeps empty lines in place so not really an issue. FYI @BFalquet @edelarua @ayogasekaram